msteams-platform/includes/tabs/content-security-policy-headers.md

```diff
@@ -1,17 +1,20 @@
 | **Microsoft 365 app host** | **frame-ancestor permission** |
-|--|--|
-| *All hosts* (New) | `*.cloud.microsoft` |
+| --- | --- |
+| All hosts (New) | `*.cloud.microsoft` |
 | Teams | `teams.microsoft.com`, `*.teams.microsoft.com` |
 | Microsoft 365 app | `*.microsoft365.com`, `*.office.com`,  |
 | Outlook | `outlook.office.com`, `outlook.office365.com`, `outlook-sdf.office.com`, `outlook-sdf.office365.com` |
-
-```http
-https://*.cloud.microsoft https://teams.microsoft.com https://*.teams.microsoft.com https://*.microsoft365.com https://*.office.com
-```
+| Copilot | `copilot.microsoft.com` |
 
 > [!WARNING]
```

Comment:
The warning note here is redundant with content in *ocdi-warning.md* include. Why don't we remove it, and then update any articles that reference this include by adding the ocdi include as well? I think it's just a couple articles in the doc set (tabs prereqs, MOS tabs tutorial).

Reply:
Done.